app/models/user_groups_users.rb

```diff
@@ -1,9 +1,10 @@
 class UserGroupsUsers < ActiveRecord::Base
+  validates_uniqueness_of :user_group_id, scope: :user_id
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.